pom.xml

```diff
@@ -77,6 +77,13 @@
             <version>1.17.1-R0.1-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
+        <!-- Paper Lib-->
+        <dependency>
+            <groupId>io.papermc</groupId>
+            <artifactId>paperlib</artifactId>
+            <version>1.0.7</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
PaperLib is not already included in paper server software, right?

Reply:
It is included in the Server Software but not in the paper dependency

Reply:
Hence we have to add it

Reply:
Yeah but if it's on the software we would not need to compile it

Reply:
Can you validate that @JT122406

Reply:
um yeah I can check on it

Reply:
only needs to be provided I changed it

Reply:
Awesome